test/TesterInternal/ClientConfig_AzureStreamProviders.xml

```diff
@@ -16,4 +16,10 @@
     <Provider Type="Orleans.Providers.Streams.SimpleMessageStream.SimpleMessageStreamProvider" Name="SMSProvider" FireAndForgetDelivery="false"/>
     <Provider Type="Orleans.Providers.Streams.AzureQueue.AzureQueueStreamProvider" Name="AzureQueueProvider" DataConnectionString="UseDevelopmentStorage=true"/>
   </StreamProviders>
+  <Assemblies>
+    <Assembly Path=".\TestGrainInterfaces.dll"/>
+    <Assembly Path=".\TestFSharpInterfaces.dll"/>
+    <Assembly Path=".\Orleans.dll"/>
+    <Assembly Path=".\TestInternalGrainInterfaces.dll"/>
+  </Assemblies>
```

Comment:
Do you actually need an entire list, or could this be just refering to the root of them, such as TesterInternal